css/src/components/form/input.scss

```diff
@@ -0,0 +1,91 @@
+$input-color: $text !default;
+$input-background-color: $body-background !default;
+$input-border-color: $text !default;
+$input-placeholder-color: $text-subtle !default;
+
+$input-hover-color: $text !default;
+$input-hover-border-color: $primary !default;
+$input-focus-border-color: $primary !default;
+
+$input-disabled-color: $text-subtle !default;
+$input-disabled-background-color: $body-background-medium !default;
+$input-disabled-border-color: $table-border-dark !default;
+
+$input-error-border-color: $danger !default;
+$input-warning-border-color: $warning-active !default;
+$input-success-border-color: $score-high !default;
+
+.input {
+	@include control;
+
+	width: 100%;
```

Comment:
This is something I never noticed about input, but seems like `input.width-full` is much more normal to me than `input.input-narrow`. Naturally this change would would make migration quite a bit trickier, but honestly, a full width input seems like a strange move to me.

Reply:
Hmm, I'll take a look into docs and how things are done there to confirm the approach. If I'm not mistaken I saw only one instance of `.input.input-narrow`, that way it would make more sense to keep it as is I think, but need to double check.

Reply:
I checked docs and there are just a few (3 to be specific) places where input's width was reset to auto so I think it makes sense to keep it full width by default. In my experience that's exactly what I see everywhere, that way form looks more organized rather than when fields have different (or just very small) widths or it is a lot of white space around. Idk, I can collect more opinions on this if you feel strongly about it.

Reply:
Okay, sounds good to me